README.md

```diff
@@ -1,110 +1,231 @@
-<br/>
 
 <p align="center">
   <a href="https://gear-tech.io">
-    <img src="https://gear-tech.io/images/logo-black.svg" width="240" alt="GEAR">
+    <img src="images/footer-logo.svg" width="400" alt="GEAR">
   </a>
 </p>
 
+<h1 align="center"> 
+The most advanced smart-contract engine
+</h1>
+
 <p align="center">
-  <b>Computational Component of Polkadot Network</b>
+ <img src="images/gear.gif" alt="Snow"><br>
```

Comment:
Not sure about the 30 MiB video on the top of README. Anyway, should we specify its width, aren't we?

Reply:
we could but why? it is not wider than its resolution.
you can check it here: https://github.com/AndrePanin/gear/blob/main/README.md